src/controllers/bookController.js

```diff
@@ -116,8 +118,12 @@ const getBook = async (req, res, next) => {
 
 const createBook = async (req, res, next) => {
   try {
-    const { coverImageUrl, ...bookData } = req.body;
-
+    const { coverImageUrl, isbn10, isbn13, ...bookData } = req.body;
+    
+    if(!isbn10 && !isbn13){
```

Comment:
please format this like it is in savedBooksController. Also, we could build a reuseable function if we do this a lot.